source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1357,21 +1360,25 @@ namespace RendererQml
 			<< "{" << "selectedDate = d.toLocaleString(Qt.locale(\"en_US\"),\"yyyy-MM-dd\");" << "}"
 			<< "else { selectedDate = '' };" << "}");
 
-		uiDateInput->Property("onTextChanged", Formatter() << "{" << uiDateInput->GetId() << "TextChanged(text);" << "setValidDate(text);" << "}");
+        uiDateInput->Property("onTextChanged", Formatter() << "{"
+            << "textChanged" << uiDateInput->GetId() << "(text);"
+            << "setValidDate(text);" 
+            << "}");
 
 		if (!input->GetValue().empty() && Utils::isValidDate(input->GetValue()))
 		{
 			uiDateInput->Property("text", Formatter() << Utils::GetDate(input->GetValue()) << ".toLocaleString(Qt.locale(\"en_US\")," << "\"" << StringDateFormat << "\"" << ")" );
 		}
 
-		uiDateInput->Property("placeholderText", Formatter() << (!input->GetPlaceholder().empty() ? input->GetPlaceholder() : "Select date") << " in " << Utils::ToLower(StringDateFormat), true);
 		uiDateInput->Property("validator", Formatter() << "RegExpValidator { regExp: " << DateRegex << "}");
 		uiDateInput->Property("onFocusChanged", Formatter() << "{" << "if(focus===true) inputMask=\"" << inputMask << "\";" <<
 			"if(focus === false){ z=0;" << "if(text === \""<< std::string(dateSeparator) + std::string(dateSeparator) << "\"){ inputMask = \"\" ; } " <<"if( " << calendar_box_id << ".visible === true){ " << calendar_box_id << ".visible=false}}} ");
         calendarTag->Property("onReleased", Formatter() << "{parent.visible=false; " << input->GetId() << ".text=selectedDate.toLocaleString(Qt.locale(\"en_US\")," << "\"" << StringDateFormat << "\")}");
 
 		calendarBoxTag->Property("Component.onCompleted", "{ Qt.createQmlObject('" + calendarTag->ToString() + "'," + calendar_box_id + ",'calendar')}");
-		uiDateInput->AddChild(calendarBoxTag);
+
+        uiDateInput->Property("placeholderText", Formatter() << (!input->GetPlaceholder().empty() ? input->GetPlaceholder() : "Select date") << " in " << Utils::ToLower(StringDateFormat), true);
+        uiDateInput->AddChild(calendarBoxTag);
```

Comment:
StringDateFormat for example holds dd/MM/yyyy
The Utils::ToLower() called while setting placeholderText uses pass by reference, because of which onFocusChanged property was having issues